src/main/java/com/amihaiemil/zold/Wallet.java

```diff
@@ -0,0 +1,32 @@
+package com.amihaiemil.zold;
+
+/**
+ * Zold network entry point.
```

Comment:
@ammaratef45 ``Zold wallet.`` here, it is not the entry point anymore.

Reply:
👍